woocommerce-myparcel.php

```diff
@@ -191,21 +204,21 @@ public function includes()
         /**
          * Instantiate classes when WooCommerce is activated
          */
-        public function load_classes()
+        public function load_classes(): void
         {
-            if ($this->is_woocommerce_activated() === false) {
-                add_action('admin_notices', [$this, 'need_woocommerce']);
+            if (! empty($this->checkInstalledWooCommerceVersion())) {
```

Comment:
Waarom returned dit iets wat wel of niet leeg kan zijn? Doe dan net als eronder, met een `boolean`

Reply:
done